packages/block-library/src/gallery/save.js

```diff
@@ -40,10 +41,16 @@ export default function save( { attributes } ) {
 							break;
 					}
 
+					let alt = image.alt || image.caption;
+
+					if ( alt && alt !== __unstableStripHTML( alt ) ) {
+						alt = __unstableStripHTML( alt );
+					}
```

Comment:
Let's maybe add a comment why we're _also_ sanitizing `image.alt` (and not just `image.caption`). Something like:

```suggestion
					// image.caption comes from a RichText component, so it can contain
					// HTML that we need to strip. In GB 9.2, image.alt would fall back to
					// image.caption without stripping the HTML, so we also run it through
					// __unstableStripHTML here.
					let alt = image.alt || image.caption;

					if ( alt && alt !== __unstableStripHTML( alt ) ) {
						alt = __unstableStripHTML( alt );
					}
```

(Feel free to change the wording if it's not clear enough.)